app/controllers/users_controller.rb

```diff
@@ -16,5 +16,24 @@ def create
     end
   end
 
-  def dashboard; end
+  def dashboard
+    # placeholder conditional for sprint
+    if current_user
+      @user_friends = BackEndFacade.get_user_friends(current_user.id)
+      @user_gyms = BackEndFacade.get_user_gyms(current_user.id)
+      @user_events = BackEndFacade.get_user_events(current_user.id)
+    end
```

Comment:
`BackEndFacade.get_user_gyms(current_user.id)`: Might consider calling the user gym memberships endpoint for this one.

Reply:
[Issue](https://github.com/tvaroglu/spot_me_frontend/issues/19) opened